aries-site/src/pages/whats-new.mdx

```diff
@@ -52,6 +52,29 @@ regarding either via [Slack](https://hpe.slack.com/archives/C04LMJWQT).
 
 <!-- BEGIN: What's New Entries -->
 
+
+## January 24 - Febuary 4, 2022
+
+### Communications
+- **Office Hours** - It was decided that the India office hours will be held every other Thursdays, 9:00 PM IST / 7:30 AM PST. This office hours is open to all. Contact us if you would like an invite.
+
+### Roadmap Updates 
+**Page Layouts** 
+ - Completed [2 column dashboard layout](https://design-system.hpe.design/templates/dashboards#two-column-dashboard) on Design System site.
+ - Started implementation on 3 column dashboard layout.
+**No Data** 
+ - Added guidance as well as an example on how to display an empty value in a [NameValueList](https://design-system.hpe.design/components/namevaluelist#name-value-pair-with-empty-value).
+**Navigation** 
+ - Published all of the different type of navigations on the [Design System](https://design-system.hpe.design/templates/navigation) site.
+
+### Stay current with the latest changes 
+- Added `position` prop to Toast to be aligned with [placement guidance](https://design-system.hpe.design/templates/toast-notifications#top-right-aligned-toast).
+- Updated visual elements for each of the newly defined core principles of accessibility: perceivable, operable, understandable and robust; view the changes in our [accessibility page](https://design-system.hpe.design/foundation/accessibility).
+- Added [Button ordering](https://design-system.hpe.design/components/button#button-ordering) based on their alignment.
+- Added a new Text color for `text-weak` 
```

Comment:
We should be explaining what the change was and why it was made. Is this a breaking change? When will it be published to the HPE theme (i.e. published to NPM)?

Reply:
Ill go into detail in the next commit